packages/paste-website/src/pages/foundations/typography/index.mdx

```diff
@@ -0,0 +1,692 @@
+
+<Card>
+  <InlineCode>Inline Code (default)</InlineCode>
+  <br />
+  <br />
+  <InlineCode>Font-family: Twilio Sans Mono</InlineCode>
+  <br />
+  <InlineCode>Font-weight: normal</InlineCode>
+  <br />
+  <InlineCode>Font-size: inherit</InlineCode>
+  <br />
+  <InlineCode>Line-height: inherit</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text-primary-stronger</InlineCode>
+  <br />
+  <InlineCode>Background-color: color-background-primary-weakest</InlineCode>
+  <br />
+  <InlineCode>Border-color: color-border-primary-weaker</InlineCode>
+  <br />
+  <Separator orientation="horizontal" verticalSpacing="space40" />
+  <InlineCode variant="minimal">Inline Code (minimal)</InlineCode>
+  <br />
+  <br />
+  <InlineCode>Font-family: Twilio Sans Mono</InlineCode>
+  <br />
+  <InlineCode>Font-weight: normal</InlineCode>
+  <br />
+  <InlineCode>Font-size: inherit</InlineCode>
+  <br />
+  <InlineCode>Line-height: inherit</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text-weak</InlineCode>
+  <br />
+  <InlineCode>Background-color: color-background-weak</InlineCode>
+  <br />
+  <InlineCode>Border-color: color-border-weak</InlineCode>
+  <br />
+</Card>
+<br />
+
+## Tokens and styles
+
+Typography is managed through the use of design tokens in Paste. Type tokens are pre-set style decisions for font size, font weight, or leading (line height). Below are just a few examples of the many tokens and styles you can use to compose your UI’s.
+
+### Some well-loved type tokens
+
+#### Size + leading (line-height)
+
+Leading refers to the overall space between lines and blocks of content. In product, we use line-height to determine the amount of space a line of text takes up. It’s important to pick the appropriate line height with your selected type size for better readability and processing of information quickly. If your line heights look like a railroad track, they are too open. We don't want our lines of text to look too open nor too tight.
+
+In Paste, we’ve matched our line height and font size tokens so that it’s easy for you to pair them. For example, using `$font-size-40` with `$line-height-40` will give you the ideal sizing and spacing pair. The same is true for `$font-size-10` and `$line-height-10`, etc.
+
+<Callout variant="neutral">
+  <CalloutHeading>Heads up about font sizes in Paste</CalloutHeading>
+  <CalloutText>
+    If you aren't using the Theme.Provider component, you must set font-size: 100%; on your page's html tag for the
+    font-sizes to be sized correctly as 1rem=16px.
+  </CalloutText>
+</Callout>
+<br />
+
+[Check out our line height and font size tokens.](/foundations/colors/#text-and-icon-colors)
+
+<Card>
+  <Text fontSize="fontSize80" lineHeight="lineHeight80">
+    Create a function to respond to Twilio Webhook Events like an incoming call.
+  </Text>
+  <br />
+  <InlineCode>Font-size: 28px</InlineCode>
```

Comment:
question: should these be tokens or pixel sizes?

Reply:
@libertycarlin

Reply:
We can do tokens to match the others